src/views/LostItems/LostItemsListContainer/LostItemsListContainer.js

```diff
@@ -13,6 +13,7 @@ import {
   Paneset,
   Button,
   Icon,
+  Callout,
```

Comment:
One suggestion I can make is to use the `CalloutContext` that is exported from `stripes-core` - rather than importing `Callout` directly from stripes-components and setting up a ref, etc. It will help your callout code scale well and keep your messages in close proximity to the code that requires them.

Reply:
Thank you!
Fixed